### DISCUSSION TITLE
Need Help with Automatic1111 not working after crash with Deforumation

### DISCUSSION THREAD

**Discussion by Slycooper808** [Rakile/deforumation#12] [Q&A]
Everything has worked and been fine so far.
Recently when trying to use deforumation, it would get stuck processing one image, so i would go to interrupt but it wouldn't affect anything. And then after I restart everything i'm not able to render anything, even txt2img, which shouldn't need deforumation to work, right?

So i uninstalled everything, automatic1111, deforum, deforumation, and then reinstall. And everything going smoothly, and then it happened again. 

How do i at the very least, get back to using txt2img and deforum, without having to uninstall and reinstall everything?

Anyways, here is the error I get :

"NansException: A tensor with all NaNs was produced in Unet. This could be either because there's not enough precision to represent the picture, or because your video card does not support half type. Try setting the "Upcast cross attention layer to float32" option in Settings > Stable Diffusion or using the --no-half commandline argument to fix this. Use --disable-nan-check commandline argument to disable this check."

**Comment by lainollll** [comment]
This is not a problem that is strictly caused by deforumation. Google
"NansException:
A tensor with all NaNs was produced in Unet. This could be either because
there's not enough precision to represent the picture, or because your
video card does not support half type.".  And you will find many reasons
for this error message and some fixes.  I to have this problem from time to
time.  Depending on what chkt file / installation / xformers / torch /
Cuda  version.   This is how it is with open source cutting edge software.
What I do to avoid it is to keep the resolution to max 640x320 when
rendering.   With the latest updated version of automatic1111 I have not
been able to reproduce the problem.

Den ons 3 maj 2023 05:48Slycooper808 ***@***.***> skrev:
